guides/micronaut-prometheus/java/src/test/java/example/micronaut/MetricsTest.java

```diff
@@ -0,0 +1,55 @@
+package example.micronaut;
+
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@MicronautTest
+public class MetricsTest {
+
+    @Client("/")
+    @Inject
+    HttpClient httpClient;
+
+    @Test
+    void metricsEndpointsIsExposed() {
+        HttpResponse<Metrics> response = httpClient.toBlocking().exchange("/metrics", Metrics.class);
+        Assertions.assertEquals(HttpStatus.OK, response.getStatus());
```

Comment:
should we standardize on static imports for assert statements?

Reply:
do you prefer static imports?

Reply:
yes, definitely for JUnit and in general for constants